src/containers/template-24-col-grid/Template.tsx

```diff
@@ -10,6 +10,16 @@ AntD Components:
 https://ant.design/components/overview/
 */
 
+const GridContentContainer = styled.div`
+  padding: 24px;
+`;
+
+const ExampleCol = styled(Col)`
+  border: 1px #1890ff solid;
```

Comment:
Can we access our variables here?

Reply:
You'd have to pull them into a ts file, not a less file.

Reply:
But, you should be using constants.